views/plugin/electron/electronmemoryconfig.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <h5>Memory</h5>
+  <div class="form-group row flex-shrink-0">
+    <label class="col-form-label col-2">Max MB</label>
+    <input class="form-control col-2" id="maxMemory" type="number" min="512" step="1"
+        ng-model="ctrl.maxMemory" ng-change="ctrl.update()" required >
```

Comment:
- I would wait to update the value until the user clicks the apply/restart button, and maybe flip that to "Apply and Restart". Currently it saves on input change, which may be bad if the user types in a value and closes Settings (not intending to change it).
- There should also be a `max` value set to the system's max available memory.
- If the value is invalid (too small or large), display an error message telling them what the valid range is and disable the apply button.

Reply:
Implemented everything suggested.